deploy/cluster-manager/config/rbac/cluster_role.yaml

```diff
@@ -20,10 +20,9 @@ rules:
     - "placement-controller-sa-kubeconfig"
     - "work-controller-sa-kubeconfig"
     - "external-hub-kubeconfig"
-# addon manager needs this to sign the customized type csr
 - apiGroups: [""]
   resources: ["secrets"]
-  verbs: ["create", "get"]
+  verbs: ["create"]
```

Comment:
we do not need create I think?

Reply:
we need create secret if clustermanager is in hosted mode. https://github.com/open-cluster-management-io/ocm/blob/9cae24c7fe2df16f7d796fa408e91b80facb0b3e/pkg/operator/operators/clustermanager/controllers/clustermanagercontroller/clustermanager_controller.go#L304